.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: ruby
+  rvm:
+    - 2.0.0
+    - 2.1.5
```

Comment:
you can remove `2.1.5`, 1) you're not using it in production 2) i doubt you plan to 3) it'll make your builds run faster.